### PR TITLE
Update tuya.ts

### PR DESCRIPTION
**Pull request by Waterbrain**
TZE204_zougpkpy

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -773,13 +773,14 @@ const definitions: Definition[] = [
             e.binary('preheat', ea.STATE, true, false).withDescription('Indicates sensor preheat is active'),
         ],
         whiteLabel: [
-            tuya.whitelabel('TuYa', 'DY-RQ500A', 'Gas sensor', ['_TZE204_zougpkpy']),
+            tuya.whitelabel('DYGSM', 'DY-RQ500A', 'Gas sensor', ['_TZE204_zougpkpy']),
         ],
         meta: {
             tuyaDatapoints: [
-                [1, 'gas', tuya.valueConverter.trueFalseEnum0],
+                [1, 'gas_detection_state', tuya.valueConverterBasic.lookup({'alarm': tuya.enum(0), 'normal': tuya.enum(1)})],
```
Comment (Koenkk) [inline src/devices/tuya.ts:780]: I propose to keep `[1, 'gas', tuya.valueConverter.trueFalseEnum0],`, it is easier to handle with `true`/false` than `alarm`/`normal`.

Comment (Waterbrain) [reply src/devices/tuya.ts:780]: But:
```
        e.enum('gas_detection_state', ea.STATE, [true, false]).withDescription('State of sensor'),

            [1, 'gas_detection_state', tuya.valueConverterBasic.lookup({True: tuya.enum(0), False: tuya.enum(1)})],

```
Captura de Pantalla 2023-10-30 a las 8 37 05

Captura de Pantalla 2023-10-30 a las 8 40 10

**Comment by Waterbrain** [comment]
```
            [1, 'gas', tuya.valueConverterBasic.lookup({True: tuya.enum(0), False: tuya.enum(1)})],
```

**Comment by Waterbrain** [comment]
Can I see what it looks like?

**Comment by Waterbrain** [comment]
Koen: 
I go to the link you sent me to edit and I see that nothing has changed.
Thanks.

**Comment by Waterbrain** [comment]
Koen:
I think I have managed to eliminate the syntax errors.
Ufffffff.
Thank you very much.

**Comment by Koenkk** [comment]
Instead of `[1, 'gas', tuya.valueConverterBasic.lookup({True: tuya.enum(0), False: tuya.enum(1)})],` can you try `[1, 'gas', tuya.valueConverter.trueFalseEnum0],` ?

**Comment by Waterbrain** [comment]
I'm sorry, but inexplicably it doesn't work. I know it should be the same, but....

**Comment by Waterbrain** [comment]
I'm sorry, but inexplicably it doesn't work. I know it should be the same, but....

**Comment by Waterbrain** [comment]
Cuando acabemos con esto. podremos arreglar esto?
Captura de Pantalla 2023-10-31 a las 9 19 02
The battery status does not work.

**Comment by Koenkk** [comment]
> I'm sorry, but inexplicably it doesn't work. I know it should be the same, but....

What doesn't work, can you explain it a bit more?

**Comment by Koenkk** [comment]
Could you provide the debug log when using `tuya.valueConverter.trueFalseEnum0` and gas is triggered on the device?

**Comment by Waterbrain** [comment]
How do I do it?

**Comment by Waterbrain** [comment]
I see that it is already supported. You could have put the image of the European. I am from Biescas and I live in Teruel.

**Comment by Waterbrain** [comment]
Three tests:

One:

```
[1, 'gas', tuya.valueConverter.trueFalseEnum0],
```
Output:

```
Debug 2023-11-03 08:25:51Received MQTT message on 'zigbee2mqtt//set' with data '{"self_test":true}'
Debug 2023-11-03 08:25:51Publishing 'set' 'self_test' to ''
Info 2023-11-03 08:25:51MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:25:51Received Zigbee message from '', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":1,"dp":8}],"seq":0}' from endpoint 1 with groupID 0
Info 2023-11-03 08:25:51MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":138,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:25:51Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":1}],"seq":52994}' from endpoint 1 with groupID 0
Info 2023-11-03 08:25:51MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":138,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:25:51Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":1,"dp":8}],"seq":53250}' from endpoint 1 with groupID 0
Info 2023-11-03 08:25:51MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":138,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:25:51Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":9}],"seq":53506}' from endpoint 1 with groupID 0
Info 2023-11-03 08:25:51MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"checking","silence":false}'
Debug 2023-11-03 08:25:56Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":4,"dp":1}],"seq":53762}' from endpoint 1 with groupID 0
Info 2023-11-03 08:25:56MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":138,"preheat":true,"self_test":true,"self_test_result":"checking","silence":false}'
Debug 2023-11-03 08:25:56Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[2],"type":"Buffer"},"datatype":4,"dp":9}],"seq":54018}' from endpoint 1 with groupID 0
Info 2023-11-03 08:25:56MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:25:58Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":1,"dp":8}],"seq":54274}' from endpoint 1 with groupID 0
Info 2023-11-03 08:25:58MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":145,"preheat":true,"self_test":false,"self_test_result":"failure","silence":false}'
```

Two:

```
[1, 'gas', tuya.valueConverterBasic.lookup({True: tuya.enum(0), False: tuya.enum(1)})],
```

Output:

```
Debug 2023-11-03 08:29:44Received MQTT message on 'zigbee2mqtt//set' with data '{"self_test":true}'
Debug 2023-11-03 08:29:44Publishing 'set' 'self_test' to ''
Info 2023-11-03 08:29:44MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":145,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:29:44Received Zigbee message from '', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":1,"dp":8}],"seq":0}' from endpoint 1 with groupID 0
Info 2023-11-03 08:29:44MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":false,"gas_detection_state":"False","gas_value":0,"linkquality":145,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:29:44Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":1}],"seq":54530}' from endpoint 1 with groupID 0
Info 2023-11-03 08:29:44MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"True","gas_detection_state":"False","gas_value":0,"linkquality":145,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:29:44Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":1,"dp":8}],"seq":54786}' from endpoint 1 with groupID 0
Info 2023-11-03 08:29:44MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"True","gas_detection_state":"False","gas_value":0,"linkquality":138,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:29:45Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":9}],"seq":55042}' from endpoint 1 with groupID 0
Info 2023-11-03 08:29:45MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"True","gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"checking","silence":false}'
Debug 2023-11-03 08:29:49Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":4,"dp":1}],"seq":55298}' from endpoint 1 with groupID 0
Info 2023-11-03 08:29:49MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"False","gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"checking","silence":false}'
Debug 2023-11-03 08:29:49Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[2],"type":"Buffer"},"datatype":4,"dp":9}],"seq":55554}' from endpoint 1 with groupID 0
Info 2023-11-03 08:29:49MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"False","gas_detection_state":"False","gas_value":0,"linkquality":138,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:29:51Received Zigbee message from '', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":1,"dp":8}],"seq":55810}' from endpoint 1 with groupID 0
Info 2023-11-03 08:29:51MQTT publish: topic 'zigbee2mqtt/', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"False","gas_detection_state":"False","gas_value":0,"linkquality":145,"preheat":true,"self_test":false,"self_test_result":"failure","silence":false}'
```

Three:

```
[1, 'gas', tuya.valueConverterBasic.lookup({true: tuya.enum(0), false: tuya.enum(1)})],
```

Output:

```
Debug 2023-11-03 08:53:20Received MQTT message on 'zigbee2mqtt/Detector de gas/set' with data '{"self_test":true}'
Debug 2023-11-03 08:53:20Publishing 'set' 'self_test' to 'Detector de gas'
Info 2023-11-03 08:53:20MQTT publish: topic 'zigbee2mqtt/Detector de gas', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"false","gas_detection_state":"False","gas_value":0,"linkquality":145,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:53:20Received Zigbee message from 'Detector de gas', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":1,"dp":8}],"seq":0}' from endpoint 1 with groupID 0
Info 2023-11-03 08:53:20MQTT publish: topic 'zigbee2mqtt/Detector de gas', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"false","gas_detection_state":"False","gas_value":0,"linkquality":145,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:53:20Received Zigbee message from 'Detector de gas', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":1}],"seq":63746}' from endpoint 1 with groupID 0
Info 2023-11-03 08:53:20MQTT publish: topic 'zigbee2mqtt/Detector de gas', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"true","gas_detection_state":"False","gas_value":0,"linkquality":138,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:53:20Received Zigbee message from 'Detector de gas', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":1,"dp":8}],"seq":64002}' from endpoint 1 with groupID 0
Info 2023-11-03 08:53:20MQTT publish: topic 'zigbee2mqtt/Detector de gas', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"true","gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:53:21Received Zigbee message from 'Detector de gas', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":4,"dp":9}],"seq":64258}' from endpoint 1 with groupID 0
Info 2023-11-03 08:53:21MQTT publish: topic 'zigbee2mqtt/Detector de gas', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"true","gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"checking","silence":false}'
Debug 2023-11-03 08:53:25Received Zigbee message from 'Detector de gas', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":4,"dp":1}],"seq":64514}' from endpoint 1 with groupID 0
Info 2023-11-03 08:53:25MQTT publish: topic 'zigbee2mqtt/Detector de gas', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"false","gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"checking","silence":false}'
Debug 2023-11-03 08:53:25Received Zigbee message from 'Detector de gas', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[2],"type":"Buffer"},"datatype":4,"dp":9}],"seq":64770}' from endpoint 1 with groupID 0
Info 2023-11-03 08:53:25MQTT publish: topic 'zigbee2mqtt/Detector de gas', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"false","gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":true,"self_test_result":"failure","silence":false}'
Debug 2023-11-03 08:53:27Received Zigbee message from 'Detector de gas', type 'commandDataReport', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[0],"type":"Buffer"},"datatype":1,"dp":8}],"seq":65026}' from endpoint 1 with groupID 0
Info 2023-11-03 08:53:27MQTT publish: topic 'zigbee2mqtt/Detector de gas', payload '{"alarm_ringtone":"melody3","alarm_time":44,"gas":"false","gas_detection_state":"False","gas_value":0,"linkquality":142,"preheat":true,"self_test":false,"self_test_result":"failure","silence":false}'
```

**Comment by Koenkk** [comment]
I think `tuya.valueConverter.trueFalseEnum0` is missing, can you try with z2m 1.33.2?

**Comment by Waterbrain** [comment]
Do you want me to remove my script from the configuration?

```
external_converters:
  - gas_sensor.js
  ```

**Comment by Waterbrain** [comment]
I have received a new one and I have done the same tests as the old one and the behavior is exactly the same.
I have read somewhere that I do not remember now, I guess in some changelog that it was no longer necessary to edit the yaml configuration file to add external converters, it was enough to do it in the configuration panel, well, in my case it does not work.

**Comment by Koenkk** [comment]
Found the issue with `trueFalseEnum0`! Pushed a fix in https://github.com/Koenkk/zigbee-herdsman-converters/commit/297090c5ef9ba87abb396923f3248e79540484cc

**Comment by Waterbrain** [comment]
Perfect.
When I was working before I retired
I was nicknamed "treeClicks", because every time I tried an application the third click of the mouse the application would crash.
I am glad that it has served to debug your code, despite my clumsiness. Well, what do I do now? I would appreciate it if you were explicit.

Have you added this?

```
            [6, 'alarm_ringtone', tuya.valueConverterBasic.lookup({'melody1': tuya.enum(0), 'melody2': tuya.enum(1), 
                'melody3': tuya.enum(2), 'melody4': tuya.enum(3),'melody5': tuya.enum(4)})],
```
And you are going to put the European image on it?

Captura de Pantalla 2023-10-11 a las 12 40 03

I also have this device connected: https://www.zigbee2mqtt.io/devices/AM43-0.45_40-ES-EB.html It has a sunlight sensor to better control the automations. The problem is that Moes has its own application that is a copy of Tuya's, although I have not been able to link it with Tuya's app. And I do not know if they have any development platform to be able to data-points and what it exposes.
What, shall we get to it?

**Comment by Waterbrain** [comment]
Sorry I had not seen this

Captura de Pantalla 2023-11-09 a las 9 22 26

**Comment by Waterbrain** [comment]
image0


I'm very sorry, it turns out that what I had interpreted as a sensor is a mini solar panel that serves to power it.
We forget about the case and sorry for my mistake and enthusiasm.

**Comment by Koenkk** [comment]
What is the datapoint of the battery?

**Comment by Waterbrain** [comment]
I already have all the console data, shall we open another thread?

**Comment by Koenkk** [comment]
thats fine

**Comment by Waterbrain** [comment]
Do you do it yourself and send me the link?

**Comment by Koenkk** [comment]
I don't know what you mean, could you provide me the dp number of battery?

**Comment by Waterbrain** [comment]
Screenshots:

Captura de Pantalla 2023-11-09 a las 11 06 10









Captura de Pantalla 2023-11-10 a las 14 22 10

```
control	Enum
{
  "range": [
    "open",
    "stop",
    "close",
    "continue"
  ]
}
percent_control	Integer	
{
  "unit": "%",
  "min": 0,
  "max": 100,
  "scale": 0,
  "step": 1
}
control_back_mode	Enum	
{
  "range": [
    "forward",
    "back"
  ]
}
```



Captura de Pantalla 2023-11-10 a las 14 23 32


```
control	Enum	
{
  "range": [
    "open",
    "stop",
    "close",
    "continue"
  ]
}
percent_control	Integer	
{
  "unit": "%",
  "min": 0,
  "max": 100,
  "scale": 0,
  "step": 1
}
percent_state	Integer	
{
  "unit": "%",
  "min": 0,
  "max": 100,
  "scale": 0,
  "step": 1
}
control_back_mode	Enum	
{
  "range": [
    "forward",
    "back"
  ]
}
work_state	Enum	
{
  "range": [
    "opening",
    "closing"
  ]
}
situation_set	Enum	
{
  "range": [
    "fully_open",
    "fully_close"
  ]
}
fault	Bitmap	
{
  "label": [
    "motor_fault"
  ]
}
```
data points:

```
{"1":"Control","2":"Curtain position setting","3":"Current curtain position","5":"Motor Direction","7":"WorkState","11":"Situation_set","12":"Fault","101":"jolcon","102":"reset","103":"ulimitset","104":"Limitset","105":"SPEED"}
```
Does not exist.....
But in your integration it comes out with the sign of ?
I have been researching in the master repository, in the Moess converters, that although you are supporting it, I can't find anything about it.

**Comment by Koenkk** [comment]
Is this device mains powered?

**Comment by Waterbrain** [comment]
Both, charger and/or solar panel

IMG_0691

**Comment by Koenkk** [comment]
Since it doesn't have a battery data point, I will set the power source to mains, this will fix the `Power` in the frontend.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by Koenkk** [comment]
Ah sorry, fixed it for TS0601_cover_4 now.

**Comment by Waterbrain** [comment]
About:

Captura de Pantalla 2023-11-13 a las 21 56 52

Power:

Captura de Pantalla 2023-11-13 a las 21 57 14

MOES especifications:

IMG_0691

and for  TuYa TS0601_gas_sensor_2

Captura de Pantalla 2023-10-11 a las 12 40 03


Please, please put the European

**Comment by Koenkk** [comment]
- Are you running the latest edge version? zigbee-herdsman-converters in settings -> about should be at least 15.115.0

> and for TuYa TS0601_gas_sensor_2

Should this be the picture for that device instead of 
TS0601_gas_sensor_2

Your picture doesn't look like a gas sensor.

**Comment by Waterbrain** [comment]
I don't know
Captura de Pantalla 2023-11-15 a las 8 10 29

```
debug 2023-11-03 09:40:53: Loaded state from file /config/zigbee2mqtt/state.json
info  2023-11-03 09:40:53: Logging to console and directory: '/config/zigbee2mqtt/log/2023-11-03.09-40-53' filename: log.txt
debug 2023-11-03 09:40:53: Removing old log directory '/config/zigbee2mqtt/log/2023-11-02.06-33-09'
info  2023-11-03 09:40:53: Starting Zigbee2MQTT version 1.33.2 (commit #unknown)
info  2023-11-03 09:40:53: Starting zigbee-herdsman (0.21.0)
debug 2023-11-03 09:40:53: Using zigbee-herdsman with settings: '{"adapter":{"concurrent":null,"delay":null,"disableLED":false},"backupPath":"/config/zigbee2mqtt/coordinator_backup.json","databaseBackupPath":"/config/zigbee2mqtt/database.db.backup","databasePath":"/config/zigbee2mqtt/database.db","network":{"channelList":[11],"extendedPanID":[221,221,221,221,221,221,221,221],"networkKey":"HIDDEN","panID":6754},"serialPort":{"path":"/dev/serial/by-id/usb-Silicon_Labs_Sonoff_Zigbee_3.0_USB_Dongle_Plus_0001-if00-port0"}}'
info  2023-11-03 09:40:54: zigbee-herdsman started (resumed)
info  2023-11-03 09:40:54: Coordinator firmware version: '{"meta":{"maintrel":1,"majorrel":2,"minorrel":7,"product":1,"revision":20221226,"transportrev":2},"type":"zStack3x0"}'
debug 2023-11-03 09:40:54: Zigbee network parameters: {"channel":11,"extendedPanID":"0x00124b0024c29ace","panID":6754}
info  2023-11-03 09:40:54: Currently 41 devices are joined:```

**Comment by Koenkk** [comment]
I forgot to push the power soruce fix for TS0601_cover_4, pushed that now.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Pull request merged**